src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -4,41 +4,52 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 public class StreamStudy {
 
-    public static long countWords() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
-    }
-
-    public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
-        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        // TODO 이 부분에 구현한다.
-    }
-
-    public static List<Integer> doubleNumbers(List<Integer> numbers) {
-        return numbers.stream().map(x -> 2 * x).collect(Collectors.toList());
-    }
-
-    public static long sumAll(List<Integer> numbers) {
-        return numbers.stream().reduce(0, (x, y) -> x + y);
-    }
-
-    public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
-    }
+	public static long countWords() throws IOException {
+		String contents = new String(Files.readAllBytes(Paths
+			.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+
+		long count = 0;
+		for (String w : words) {
+			if (w.length() > 12)
+				count++;
+		}
+		return count;
+	}
+
+	public static void printLongestWordTop100() throws IOException {
+		String contents = new String(Files.readAllBytes(Paths
+			.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+
+		words.stream()
+			.sorted(Comparator.comparing(String::length).reversed())
+			.distinct()
+			.filter(word -> word.length() > 12)
+			.map(String::toLowerCase)
+			.limit(100)
+			.forEach(System.out::println);
+	}
+
+	public static List<Integer> doubleNumbers(List<Integer> numbers) {
+		return numbers.stream().map(x -> 2 * x).collect(Collectors.toList());
+	}
```

Comment:
```suggestion
	public static List<Integer> doubleNumbers(List<Integer> numbers) {
		return numbers.stream()
				.map(x -> 2 * x)
				.collect(Collectors.toList());
	}
```

개행으로 가독성을 높여보면 좋을 것 같아요 :)